python/tskit/trees.py

```diff
@@ -7160,6 +7160,30 @@ def count_topologies(self, sample_sets=None):
 
         yield from combinatorics.treeseq_count_topologies(self, sample_sets)
 
+    def find_ibd(self, samples, max_time=None, min_length=None):
+        """
+        A wrapper for :meth:`TableCollection.find_ibd`. See this method for more info.
```

Comment:
Most methods have the TreeSequence method the "primary" one, I think - perhaps the main docs should live here?

Reply:
Good point! I don't have strong feelings either way, I'm happy to shift the main docs over there. Will update the PR in the afternoon.